modules/market/market.models.ts

```diff
@@ -1,10 +1,107 @@
 import * as mongo from '../../util/mongo';
 import type * as Types from './market.types';
 
-export const Market = mongo.createModel<Types.MarketDocument>('Market', {
-  value: { type: String, required: true },
+export const MarketAnalysis = mongo.createModel<Types.MarketAnalysisDocument>('MarketAnalysis', {
+  action: { type: String, required: true },
+  asset: { type: String, required: true },
+  signalType: { type: String, required: true },
+  confidence: { type: Number, required: true },
+  riskLevel: { type: String, required: true },
+  timeFrame: { type: String, required: true },
+  source: { type: String, required: true },
+  summary: { type: String, required: true },
+  labels: {
+    sector: { type: String },
+    topic: { type: String },
+    sentiment: { type: String },
+  },
 });
 
-export const MarketPair = mongo.createModel<Types.MarketPairDocument>('MarketPair', {
-  value: { type: String, required: true },
+export const MarketInvestor = mongo.createModel<Types.MarketInvestorDocument>('MarketInvestor', {
+  name: { type: String, required: true },
+  portfolioIds: { type: mongo.Schema.Types.ObjectId, required: true },
```

Comment:
plural? is this suppose to be an array?

Reply:
i was thinking an investor could potentially have different portfolios but i fixed this in the next commit making this an array same goes for investmentIds

a porfolio could have different investments